src/components/tf-graph-datasource/tf-graph-datasource.tsx

```diff
@@ -0,0 +1,529 @@
+                        z: 100,
+                        yAxisIndex: yAxisIndex ? yAxisIndex : null,
+                        data: errorData,
+                    }
+                    updatedTraceData.push(singleErrorTraceData);
+                    let singleTraceData = {
+                        id: traceID,
+                        data: data,
+                        type: isLineWithErrorBar ? InbuiltTraceType.LINE : InbuiltTraceType.SCATTER,
+                        lineStyle: traceType === TraceType.LINE_WITH_ERROR_BARS ? { type: this.getLineType(lineType), width: lineWidth } : '',
+                        symbol: this.getMarkerType(markerType, markerFill),
+                        symbolSize: markerType === MarkerType.ELLIPSE ? [16, 8] : [markerWidth, markerHeight],
+                        itemStyle: {
+                            color: `${traceColor}`,
+                            opacity: markerThickness
+                        },
+                        yAxisIndex: yAxisIndex ? yAxisIndex : null,                    }
+                    updatedTraceData.push(singleTraceData);
+                    // when chartType is heatmap
+                } else if (this.chartType === ChartType.HEATMAP) {
+                    let singleTraceData = {
+                        id: traceID,
+                        data: data,
+                        type: ChartType.HEATMAP,
+                        emphasis: {
+                            itemStyle: {
+                                borderColor: '#333',
+                                borderWidth: 1,
+                                shadowBlur: 10,
+                                shadowColor: 'rgba(0, 0, 0, 0.5)',
+                            }
+                        },
+                        label: {
+                            show: true
+                          },
+                    }
+                    updatedTraceData.push(singleTraceData);
+                }
+                 else {
+                    let singleTraceData = {
+                        id: traceID,
+                        data: data,
+                        type: traceType,
+                        lineStyle: traceType === InbuiltTraceType.LINE ? { type: this.getLineType(lineType), width: lineWidth } : '',
+                        symbol: this.getMarkerType(markerType, markerFill),
+                        symbolSize: markerType === MarkerType.ELLIPSE ? [16, 8] : [markerWidth, markerHeight],
+                        itemStyle: {
+                            color: `${traceColor}`,
+                            opacity: markerThickness
+                        },
+                        yAxisIndex: yAxisIndex ? yAxisIndex : null,                    
+                    }
+                    updatedTraceData.push(singleTraceData);
+                }
+            }
+        }
+        console.log("🚀 ~ file: tf-graph-datasource.tsx:461 ~ TfGraphDatasource ~ getTraceData ~ updatedTraceData", updatedTraceData)
```

Comment:
@sagarbag6132401  please remove the console log

Reply:
I have removed all the console log.